src/sentry/models/minidump.py

```diff
@@ -28,11 +29,14 @@ class MinidumpFile(Model):
     __core__ = False
 
     file = FlexibleForeignKey('sentry.File')
-    event_id = models.CharField(max_length=36, unique=True)
+    project = FlexibleForeignKey('sentry.Project', null=True)
```

Comment:
Why could this be NULL? Since we haven't written any data, we shouldn't leave it nullable. Now is the time to correctly do it.

Reply:
@mattrobenolt no reason and i will change that. I just wanted to make travis run the tests first because my local test setup gives weird errors :D